### DISCUSSION TITLE
v1.1.3

### DISCUSSION THREAD

**Discussion by chaitanyapotti** [Web3Auth/Web3Auth#354] [Announcements]
## What's Changed
* updated deps in demo apps and tested by @rinzler99 in https://github.com/Web3Auth/Web3Auth/pull/287
* Fix old starkex demo by @yashovardhan in https://github.com/Web3Auth/Web3Auth/pull/288
* fix css loader for umd and cjs builds by @himanshuchawla009 in https://github.com/Web3Auth/Web3Auth/pull/297

**Full Changelog**: https://github.com/Web3Auth/Web3Auth/compare/v1.1.2...v1.1.3

<hr /><em>This discussion was created from the release <a href='https://github.com/Web3Auth/Web3Auth/releases/tag/v1.1.3'>v1.1.3</a>.</em>

**Comment by HauserY** [comment]
###
